pyxrf/model/fileio.py

```diff
@@ -88,12 +88,62 @@ def __init__(self, **kwargs):
         self.mask_data = None
 
         # Display PyXRF version in the window title
-        global pyxrf_version
-        if pyxrf_version[0] not in 'vV':
-            pyxrf_version = f"v{pyxrf_version}"
-        self.window_title_base = f"PyXRF: X-ray Fluorescence Analysis Tool ({pyxrf_version})"
+        ver, new_ver = self._get_pyxrf_version_str()
+        ver_str = ver
+        if new_ver:
+            ver_str += f" - new version {new_ver} is available"
+        self.window_title_base = f"PyXRF: X-ray Fluorescence Analysis Tool ({ver_str})"
         self.window_title = self.window_title_base
 
+    def _get_pyxrf_version_str(self):
+        """
+        The function returns the tuple of strings:
+        - current version number of PyXRF;
+        - the latest version of PyXRF from nsls2forge channel.
+        If current version is the latest, then the second string is empty
+        """
+
+        import requests
+        from distutils.version import LooseVersion
+
+        # Determine the current version of PyXRF
+        global pyxrf_version
+        pyxrf_version_str = pyxrf_version
+        if pyxrf_version_str[0] not in 'vV':
```

Comment:
```suggestion
        if not pyxrf_version_str.lower().startswith('v'):
```

Reply:
I think that the first version is more computationally efficient, but the efficiency doesn't really matter here.